lib/mongo/collection/view/readable.rb

```diff
@@ -24,6 +24,19 @@ class View
       # @since 2.0.0
       module Readable
 
+        # The count options potentially included in the options hash, using the
+        # configure method.
+        #
+        # @api private
+        COUNT_OPTIONS = ["hint", "limit", "max_time_ms", "skip", "comment"]
+
+        # The aggregate options potentially included in the options hash, using the
+        # configure method.
+        #
+        # @api private
+        AGGREGATE_OPTIONS = ["allow_disk_use", "batch_size", "max_time_ms",
+          "max_await_time_ms", "comment", "hint"]
```

Comment:
We freeze arrays assigned to constants to prevent unintentional modification.

Reply:
done